pkg/prometheus/promcfg_test.go

```diff
@@ -4609,9 +4585,26 @@ remote_write:
 `,
 		},
 		{
-			version: "v2.23.0",
+			version: "v2.26.0",
```

Comment:
could you create a new test case instead? this one is about testing that `MetadataConfig` is supported when version >= v2.23.0.

Reply:
If I read correctly, that test you are mentioning is below this one added after the rebase, at https://github.com/tolleiv/prometheus-operator/blob/remote_write_sigv4/pkg/prometheus/promcfg_test.go#L4659?

The diff doesn't help here 😄

Reply:
Improved the diff in latest commit, I put the new sigv4 test at the end of `promcgf_test.go`.